WordPress/Classes/Extensions/WpApiError+Localized.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+import WordPressAPI
+
+/// `WpApiError` confirms to `LocalizedError`, but with an implementation that's not suitable for displaying on UI.
+/// When presenting `WpApiError`, we should use the `errorMessage` function instead of `localizedDescription`.
```

Comment:
We'll be able to implement `LocalizedError` properly in wordpress-rs, once/if https://github.com/mozilla/uniffi-rs/pull/2319 is merged and released.